analytic_diffuse/solutions.py

```diff
@@ -118,38 +167,54 @@ def projgauss(uvecs, a, order=30, usesmall=False, uselarge=False):
     a: float
         Gaussian width parameter.
     order: int
-        Expansion order.
+        If not `chunk_order`, the expansion order. Otherwise, this is the *maximum*
+        order of the expansion.
+    chunk_order: int
+        If non-zero, the expansion will be summed until convergence (or max order is
+        reached).
```

Comment:
It's not clear to me what chunk_order does. Does it just set the steps to take while looking for convergence?

Reply:
Yes. I've tried to document that more clearly now.